### DISCUSSION TITLE
How to make the homepage the blog?

### DISCUSSION THREAD

**Discussion by fullStackDataSolutions** [arthelokyo/astrowind#172] [Q&A]
I'm new to Astro but I've been Next for a long time. 

How do I make the blog use the homepage "/" as it's default page? 

I tried to add that in the settings, but  it still showed whatever I had on the index page.

**Comment by prototypa** [comment]
Hello @blazestudios23 

To do this just delete the home page that comes with **AstroWind** index.astro (or rename it) and then go to the **AstroWind** configuration and in the blog / list settings instead of "blog" change it to ""

Ex:

```js
const CONFIG = {
  // ..

  blog: {
     // ..

    list: {
      pathname: '', // -> This
      noindex: false,
      disabled: false,
    },

    // ..
  },
};
```

**Reply by abhinavkakku** [reply]
hey @prototypa , 
this does not works on github pages or gitlab pages.

Is there a fix ?

This has been working on local builds but redirection does not happens when I set blog as default home like you have advised above.

**Reply by abhinavkakku** [reply]
@prototypa  this does not seems to work when deployed on github-pages or gitlab pages.

I was trying on local builds, works perfectly fine,
but on github pages, the blog-list takes home path when edited as you said,
but home does not displays the blog list as intended.
just for example I have it set at `ethicalhackx.github.io`

`apps:
  blog:
    isEnabled: true
    postsPerPage: 10

    post:
      isEnabled: true
      permalink: '/%slug%' # Variables: %slug%, %year%, %month%, %day%, %hour%, %minute%, %second%, %category%
      robots:
        index: true

    list:
      isEnabled: true
      pathname: '' # Blog main path, you can change this to "articles" (/articles)
      robots:
        index: true`

**Comment by abhinavkakku** [comment]
This configuration changes works on local builds, but fails on gitlab pages.

On gitlab pages when deployed with official workflow, the blog list home page is not loaded as homepage, instead the default astrowind landing-page is loaded as homepage

**Comment by prototypa** [comment]
Hi @fullStackDataSolutions and @abhinavkakku 

Example of **AstroWind** in _github pages_:

- Demo: https://prototypa.github.io/astrowind/
- Repo: https://github.com/prototypa/astrowind

Steps:

1. In _config.yaml_ replace `apps.blog.list.pathname` to `''`
2. In _config.yaml_ replace `site.site` and `site.base` to corresponding values in _github pages_ (Ex: `https://prototypa.github.io` and `/astrowind`)
3. Add `.nojekyll` to `public/` folder
4. Create github actions to auto deploy to _github pages_

**Reply by abhinavkakku** [reply]
@prototypa thanks, suggesting here to add another step to this -> removing the default index page or renaming it ( if it is not already mentioned), Figured out doing so solved it for me.
in future releases maybe there is more clarity on this.
Thanks for solving this.
